.travis.yml

```diff
@@ -15,6 +15,8 @@ addons:
       - $(ls tmp/screenshots/*.png | tr "\n" ":")
       - $(ls tmp/csvs/*.csv | tr "\n" ":")
     debug: true
+gemfile:
+- Gemfile
```

Comment:
@rhymes I guess now that we are not using dual booting, this change is unnecessary. 😄

Reply:
yeah, I kept it as a reminder :D